### PR TITLE
[ADD] attachment_preview

### PR DESCRIPTION
**Pull request by hbrunn**
Port of version 7.0

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
Great work! Checked the changes with the 7.0 version and they all seemed warranted. Good to see that OpenOffice template files are now supported as well by ViewerJS ;-) :+1:

File: attachment_preview/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+#
+##############################################################################
+{
+    "name": "Preview attachments",
+    "version": "1.1",
+    "author": "Therp BV",
+    "license": "AGPL-3",
+    "complexity": "normal",
+    "description": """
+This addon allows to preview attachments supported by http://viewerjs.org.
```
Comment (pedrobaeza) [inline attachment_preview/__openerp__.py]: Please extract this into README.rst, and consider to use OCA new description template (https://github.com/OCA/maintainers-tools/blob/master/template/module/README.rst)

**Comment by pedrobaeza** [comment]
A little remark. The rest LGTM.

File: attachment_preview/__openerp__.py (hunk 1 above)
Comment (hbrunn) [inline attachment_preview/__openerp__.py]: Thanks for insisting, I'm still in the transition phase and tend to forget this

**Comment by pedrobaeza** [comment]
:+1:

**Comment by guewen** [comment]
Excellent! :+1:

**Pull request merged**